tests/rules/taint_object_funcdef.js

```diff
@@ -0,0 +1,20 @@
+app.get('/ok', async (req, res) => {
```

Comment:
Could we also add https://semgrep.dev/playground/s/17Qx as test `taint_object_funcdef1` ?